docker/src/main/java/brooklyn/entity/container/docker/application/DockerServiceTypeResolver.java

```diff
@@ -72,16 +57,13 @@ public String getBrooklynType(String serviceType) {
         String imageName = Iterables.get(parts, 0);
         String imageTag = Iterables.get(parts, 1, "latest");
         log.debug("Creating Docker service entity with image {} and tag {}", imageName, imageTag);
+
+        EntitySpec<VanillaDockerApplication> spec = EntitySpec.create(VanillaDockerApplication.class);
         spec.configure(DockerAttributes.DOCKER_IMAGE_NAME, imageName);
         if (parts.size() == 2) {
             spec.configure(DockerAttributes.DOCKER_IMAGE_TAG, imageTag);
         }
-        if (resolver.getAttrs().containsKey("id")) {
```

Comment:
Thanks @neykov, I agree it's not specifically needed here, but I was maybe wanting to do something more with this for Mesos and Marathon. I'll leave it for the moment, though.